flowbench/BenchNet2.actor.cpp

```diff
@@ -61,3 +62,31 @@ static void bench_net2(benchmark::State& benchState) {
 }
 
 BENCHMARK(bench_net2)->Range(1, 1 << 16)->ReportAggregatesOnly(true);
+
+ACTOR static Future<Void> benchDelayActor(benchmark::State* benchState, bool useYield) {
```

Comment:
Given the low runtime of each iteration (10s of nanoseconds), the overhead of a branch miss is non-trivial, so I think we should make `useYield` a template parameter.

Reply:
Done.